### PR TITLE
refactor: rebranding changes

### PR DESCRIPTION
**Pull request by sumitarora**

### CODE REVIEW

File: packages/@angular/cli/lib/config/schema.json [hunk 1]
```diff
@@ -336,7 +336,7 @@
           "default": true
         },
         "packageDeprecation": {
-          "description": "Show a warning when the user installed angular-cli.",
+          "description": "Show a warning when the user installed angular cli.",
```
Comment (filipesilva) [inline packages/@angular/cli/lib/config/schema.json]: Keep this one please, it does refer to the old package.

File: packages/@angular/cli/models/config.ts [hunk 2]
```diff
@@ -35,7 +35,7 @@ function getUserHome() {
 
 export class CliConfig extends CliConfigBase<ConfigInterface> {
   static configFilePath(projectPath?: string): string {
-    // Find the configuration, either where specified, in the angular-cli project
+    // Find the configuration, either where specified, in the angular cli project
```
Comment (filipesilva) [inline packages/@angular/cli/models/config.ts]: Use `Angular CLI` here.

File: packages/@angular/cli/package.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
   },
   "keywords": [
     "angular",
-    "angular-cli"
+    "angular cli"
```
Comment (filipesilva) [inline packages/@angular/cli/package.json]: Use `Angular CLI` here.

File: plugins/karma.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-// This file is necessary when using a linked angular-cli to this repo, meaning that
+// This file is necessary when using a linked angular cli to this repo, meaning that
```
Comment (filipesilva) [inline plugins/karma.js]: Use `@angular/cli` here.

**Comment by hansl** [comment]
Could you also remove the option mentioning Angular2 in the protractor config in our project template?

**Comment by sumitarora** [comment]
@filipesilva @hansl done the changes 😄

File: packages/@angular/cli/package.json [hunk 5]
```diff
@@ -9,7 +9,7 @@
   },
   "keywords": [
     "angular",
-    "angular-cli"
+    "Angular CLI"
```
Comment (hansl) [inline packages/@angular/cli/package.json]: Keep the old value as well (but add the new one) so that people can search for it and find this package.

Comment (sumitarora) [inline packages/@angular/cli/package.json]: done 😄

**Pull request merged**